<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSettingForm</class>
 <widget class="QWidget" name="SoundSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSaveandQuit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>770</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Save and Quit</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>580</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbtnMute">
      <property name="text">
       <string>Mute</string>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnFullVolumn">
      <property name="text">
       <string>FullVolume</string>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>621</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>640</y>
     <width>621</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Import BGM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
